### DISCUSSION TITLE
Hi! Is there any tutorial about follower system with this package?

### DISCUSSION THREAD

**Discussion by khrisvandal** [revsys/django-friendship#151] [Q&A]
I recently installed this package to my site to add a follower system but is not working.

This code is not giving me any error, it's just not working. If you have any experience using the django-friendship package, and know how i can do to make this work, please let me know.

I opened a question in stackoverflow about this for checking the code:
https://stackoverflow.com/questions/62258800/has-anyone-tried-django-friendship-package-problem-following-an-user

Thank you for your help!

**Comment by ibrahimsamson** [comment]
I use the package it's working just fine, I am not an expert but I suggest that you check your admin page or the DB if 
there is any change or not when you run your code

**Comment by OmegaHawkeye** [comment]
@chrislumber  Hey, I'm actually right now using it again after a long break because I was stuck and didn't understand how. But now like 2-3 months later after implementing other packages and getting a better undersanding about how they all structured their docs (README.md) similar. It's actually pretty simple. If you need help I can send you a code snippet that I implemented so u can use it and mby understand it then too.
